### ISSUE TITLE
[MNG-7191] Remove undocumented environment variable MAVEN_CONFIG in startup scripts

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#8256]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** opened **[MNG-7191](https://issues.apache.org/jira/browse/MNG-7191?redirect=false)** and commented

During a review I have found this:

```
./apache-maven/src/assembly/shared/run:MAVEN_CMD_LINE_ARGS="$MAVEN_CONFIG $@"
```

There is no counterpart for Windows command prompt nor is it used somewhere else in ITs. Let's remove it if we see no use for.

---

**Backported to:** [4.0.0-alpha-2](https://github.com/apache/maven/milestone/113?closed=1)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte), here is another one.

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

Fixed with [6b37fcbda828f1b77e01a401b09b04b55b871956](https://gitbox.apache.org/repos/asf?p=maven.git;a=commit;h=6b37fcbda828f1b77e01a401b09b04b55b871956).
